apps/frontend/src/libs/modules/toast-notification/toast-notification.component.tsx

```diff
@@ -0,0 +1,8 @@
+import { ToastContainer } from "react-toastify";
```

Comment:
This component should be in `~/libs/components` folder
Also rename to `notification`, instead of `toast-notification`